package.json

```diff
@@ -22,15 +22,20 @@
     "run-scraper": "ts-node utils/api/scrapers/_main.ts"
   },
   "dependencies": {
+    "@headlessui/react": "^1.4.2",
     "@babel/core": "^7.16.5",
     "@tailwindcss/forms": "^0.3.4",
+    "@types/lodash": "^4.14.177",
     "isarray": "^2.0.5",
     "joi": "^17.5.0",
-    "next": "^12.0.7",
+    "lodash": "^4.17.21",
+    "next": "^12.0.3",
```

Comment:
unintentional downgrade of `next`, however I did an intentional downgrade in my yarn 2 branch, so all good here!